test_api.py

```diff
@@ -2869,43 +2899,50 @@ def test_get_system_profile_ignores_culled(self):
 
 
 class QueryStalenessGetHostsIgnoresCulledTestCase(QueryStalenessGetHostsBaseTestCase, DeleteHostsBaseTestCase):
-    def test_patch_ignores_culled(self):
+    @patch("api.host.emit_patch_event")
+    def test_patch_ignores_culled(self, emit_event):
         url = HOST_URL + "/" + self.culled_host["id"]
 
         self.patch(url, {"display_name": "patched"}, 404)
 
-    def test_patch_works_on_non_culled(self):
+    @patch("api.host.emit_patch_event")
+    def test_patch_works_on_non_culled(self, emit_patch_event):
         url = HOST_URL + "/" + self.fresh_host["id"]
 
         self.patch(url, {"display_name": "patched"}, 200)
 
-    def test_patch_facts_ignores_culled(self):
+    @patch("api.host.emit_patch_event")
+    def test_patch_facts_ignores_culled(self, emit_patch_event):
         url = HOST_URL + "/" + self.culled_host["id"] + "/facts/ns1"
 
         self.patch(url, {"ARCHITECTURE": "patched"}, 400)
 
-    def test_patch_facts_works_on_non_culled(self):  # broken
+    @patch("api.host.emit_patch_event")
+    def test_patch_facts_works_on_non_culled(self, emit_patch_event):  # broken
         url = HOST_URL + "/" + self.fresh_host["id"] + "/facts/ns1"
 
         self.patch(url, {"ARCHITECTURE": "patched"}, 200)
 
-    def test_put_facts_ignores_culled(self):
+    @patch("api.host.emit_patch_event")
+    def test_put_facts_ignores_culled(self, emit_patch_event):
         url = HOST_URL + "/" + self.culled_host["id"] + "/facts/ns1"
 
         self.put(url, {"ARCHITECTURE": "patched"}, 400)
 
-    def test_put_facts_works_on_non_culled(self):  # broken
+    @patch("api.host.emit_patch_event")
+    def test_put_facts_works_on_non_culled(self, emit_patch_event):  # broken
         url = HOST_URL + "/" + self.fresh_host["id"] + "/facts/ns1"
         print(url)
         self.put(url, {"ARCHITECTURE": "patched"}, 200)
 
-    def test_delete_ignores_culled(self):
+    @patch("api.host.emit_patch_event")
+    def test_delete_ignores_culled(self, emit_patch_event):
         url = HOST_URL + "/" + self.culled_host["id"]
 
         self.delete(url, 404)
 
     @patch("lib.host_delete.emit_event", new_callable=MockEmitEvent)
```

Comment:
Let’s remove the _new_callable_, it’s not used at all.

```suggestion
    @patch("lib.host_delete.emit_event")
```